### PR TITLE
Week 2 Step 5-2: Use Datastore for storage and retrival of comments

### PR DESCRIPTION
**Pull request by astrobliss**
DataServerlet no longer locally stores comments. All comment access is now done through DataStore so that comments are persistent across server restarts.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -30,35 +30,44 @@
...
+   * Iff request has a non-null comment-text parameter,
+   *   then create and store the parameter's String as a Datastore Comment Object
    * Always redirect to index.html
    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("comment-text");
     if(comment != null){
-      comments.add(comment);
+      long timestamp = System.currentTimeMillis();
```
Comment (chrisachille) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Might want to use timestampMs just to be clear what format it is in?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -30,35 +30,44 @@
...
       Entity commentEn = new Entity("Comment");
       commentEn.setProperty("text",comment);
+      commentEn.setProperty("timestamp",timestamp);
```
Comment (chrisachille) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: timestampMs or timestamp_ms?

**Review by chrisachille — APPROVED** [review]
Couple comments, but other than that, looks awesome.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (astrobliss) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: sounds good, extra clarity never hurt anyone!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (astrobliss) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: "timestampMs" now replaces "timestamp" for the long name and attribute name.

**Pull request merged**